docs/config.json

```diff
@@ -221,6 +221,11 @@
           "file": "docs/operations/set_replica_identity.mdx"
         }
       ]
+    },
+    {
+      "title": "Release notes",
+      "href": "/releases",
+      "file": "docs/releases/README.md"
```

Comment:
I'm not sure what causes it but this new page appears with this 404 link at the bottom of the page:

<img width="510" alt="image" src="https://github.com/user-attachments/assets/a06bf220-0c30-4697-9876-ab7bc6c7f27a" />

Reply:
It happens on all previews for some reason. I did not want to bug Richard with it because it did not seem important enough. :D